db/migrations/versions/021_add_geospatial_dim_table.py

```diff
@@ -0,0 +1,44 @@
+"""Add geospatial_dim table and postcode_prefix and itl_region mapping data
+
+Revision ID: 021_add_geospatial_dim_table
+Revises: 020_add_prog_junc_to_outputs
+Create Date: 2024-03-06 16:13:06.143726
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+import core
+
+# revision identifiers, used by Alembic.
+revision = "021_add_geospatial_dim_table"
+down_revision = "020_add_prog_junc_to_outputs"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+
+    op.create_table(
+        "geospatial_dim",
+        sa.Column("id", core.db.types.GUID(), nullable=False),
```

Comment:
Can we autogenerate the IDs instead e.g. 
`sa.Column("id", postgresql.UUID(as_uuid=True), nullable=False, primary_key=True, default=uuid.uuid4),`

Reply:
I think they're normally auto-generated by the base model in `entities.py` when data is added, but in the seed data I've actually already generated uuids for this reference data so that if we come to extend the table and add more mappings it'll match the ids and merge the rows so an FK relationships are preserved.